### ISSUE TITLE
SAGA Input/output error

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#89]
In GitLab by @dmanubens on Jan 20, 2016, 18:27

```
5 of 7 jobs remaining (18:19)
[CRITICAL] Unhandled exception on Autosubmit: read from process failed '[Errno 5] Input/output error' : (  login2      16*s01      *fc0_2_SIM Jan 20 18:18
                                             32*s12
                                             32*s16
                                             50*s02
PROMPT-0->Shared connection to mn2.bsc.es closed.
) (/home/Earth/dmanuben/venvs/as_320/lib/python2.7/site-packages/saga/utils/pty_process.py +679 (read)  :  % (e, self.tail)))
Traceback (most recent call last):
  File "/home/Earth/dmanuben/pyCharm/autosubmit/autosubmit/autosubmit.py", line 286, in parse_args
    return Autosubmit.run_experiment(args.expid)
  File "/home/Earth/dmanuben/pyCharm/autosubmit/autosubmit/autosubmit.py", line 602, in run_experiment
    if job.status != job.update_status(platform.check_job(job.id)):
  File "/home/Earth/dmanuben/pyCharm/autosubmit/autosubmit/platforms/platform.py", line 427, in check_job
    if jobid not in self.service.jobs:
  File "/home/Earth/dmanuben/venvs/as_320/lib/python2.7/site-packages/saga/job/service.py", line 378, in list
    return self._adaptor.list (ttype=ttype)
  File "/home/Earth/dmanuben/venvs/as_320/lib/python2.7/site-packages/saga/adaptors/cpi/decorators.py", line 57, in wrap_function
    return sync_function (self, *args, **kwargs)
  File "/home/Earth/dmanuben/pyCharm/autosubmit/autosubmit/platforms/mn_adaptor.py", line 865, in list
    ret, out, _ = self.shell.run_sync("%s -a" % 'bjobs')
  File "/home/Earth/dmanuben/venvs/as_320/lib/python2.7/site-packages/saga/utils/pty_shell.py", line 647, in run_sync
    self.pty_shell.flush ()
  File "/home/Earth/dmanuben/venvs/as_320/lib/python2.7/site-packages/saga/utils/pty_process.py", line 174, in flush
    tmp = self.read (timeout=0.1)
  File "/home/Earth/dmanuben/venvs/as_320/lib/python2.7/site-packages/saga/utils/pty_process.py", line 679, in read
    % (e, self.tail))
NoSuccess: read from process failed '[Errno 5] Input/output error' : (  login2      16*s01      *fc0_2_SIM Jan 20 18:18
                                             32*s12
                                             32*s16
                                             50*s02
PROMPT-0->Shared connection to mn2.bsc.es closed.
) (/home/Earth/dmanuben/venvs/as_320/lib/python2.7/site-packages/saga/utils/pty_process.py +679 (read)  :  % (e, self.tail))
```

**Comment by kinow** [comment]
In GitLab by @jvegreg on Jan 21, 2016, 11:23

Unable to reproduce

**Comment by kinow** [comment]
In GitLab by @dmanubens on May 13, 2016, 16:20

fixed in 57c509cc061530a4bfd59d689fbdcfdc95af75f9
